scripts/inspect.sh

```diff
@@ -238,6 +238,27 @@ function suggest_fixes {
     printf -- "\t  https://microk8s.io/docs/troubleshooting#heading--common-issues \n"
   fi
 
+  msg_containerd=`journalctl -n 1000 -u "snap.microk8s.daemon-containerd" |grep -E 'failed.*error'|awk -F 'error=' '{print $2}'|sort -u`
+  function inspect_containerd {
+      IFS=$'\n'; 
+      pre_msg="";
+      tcps="";
+
+      for line in $1; do
+          msg=`echo $line|awk -F 'dial tcp' '{print $1}'`
+          tcp=`echo $line|awk -F 'dial tcp' '{print $2}'`
+          if [ "$pre_msg" != "$msg" ]; then
+              [ -n "$tcps" ] && ( echo "$pre_msg"|sed 's/":/"\n/g';echo $tcps; )
+              pre_msg="$msg"
+              tcps=""
+          fi
+          tcps="$tcps $tcp"
+      done
+      [ -n "$tcps" ] && ( echo "$pre_msg"|sed 's/":/"\n/g';echo $tcps; )
+  }
+  [ -n "$msg_containerd" ] && \
+    printf -- '\033[0;33mWARNING: \033[0m the containerd for MicroK8s maybe have some fail. \n' && \
```

Comment:
This does not suggest any fixes to the user. It just says that one of the components of kubernetes has some errors in the logs. There is nothing the user can do about that, no suggestion there. The user already knows there is something wrong with her cluster this is why she called `microk8s inspect`.

It would be best to detect a specific actionable error and suggest a fix/reconfiguration or point the user to a site where the error is discussed.